src/main/java/org/simulator/examples/SBMLTestSuiteRunnerWrapper.java

```diff
@@ -63,6 +61,8 @@ public static void main(String[] args) throws IOException, XMLStreamException, M
         String[] amounts = String.valueOf(properties.getProperty("amount")).split(",");
         String[] concentrations = String.valueOf(
                 properties.getProperty("concentration")).split(",");
+        double absolute = (!properties.getProperty("absolute").isEmpty()) ? Double.parseDouble(properties.getProperty("absolute")) : 0d;
+        double relative = (!properties.getProperty("relative").isEmpty()) ? Double.parseDouble(properties.getProperty("relative")) : 0d;
```

Comment:
It would be preferable to declare `String` literals in the form `public static final String ABSOLUTE = "absolute"` and then to reuse these constants where needed. This reduces maintenance effort and possible errors caused by typos. 

Another possibility is creating an internal `enum` such as:
```
public enum SettingsKeywords {
   absolute;
   ...
}
```
that can be uses similarly.

Reply:
Updated by adding final variables for Strings.